maven-xml-impl/pom.xml

```diff
@@ -26,7 +26,7 @@ under the License.
   </parent>
 
   <artifactId>maven-xml-impl</artifactId>
-  <name>Implementation of Maven API XML</name>
+  <name>maven-xml-impl(Implementation of Maven API XML)</name>
```

Comment:
This is not coherent with the other packages afaik.
I'd rather do that globally all all modules if we want to go that way.